dubbo-remoting/pom.xml

```diff
@@ -35,6 +35,7 @@
         <module>dubbo-remoting-http</module>
         <module>dubbo-remoting-netty</module>
         <module>dubbo-remoting-zookeeper</module>
+        <module>dubbo-remoting-zookeeper-curator5</module>
```

Comment:
Please add module shading support in dubbo-distribution.

Reply:
Done.